src/mantle/index.ts

```diff
@@ -242,6 +249,8 @@ export function geoSpectrum(
     crossSection
   );
 
+  const mantleAbundanceU238 = U238flux / ISOTOPIC_NEUTRINO_LUMINOSITY.U238 / mantleGeophysicalResponse
+
   const mantleHeatingU238 = (U238flux / ISOTOPIC_NEUTRINO_LUMINOSITY.U238 / mantleGeophysicalResponse) * ISOTOPIC_DECAY_HEATING.U238 * mantleMass
```

Comment:
I would make this a function of the mantleAbundance rather than repeating the calculation for that:

```javascript
 const mantleHeatingU238 = mantleAbundanceU238 * ISOTOPIC_DECAY_HEATING.U238 * mantleMass
```